aws/templates/docker_compose_xchain.tftpl

```diff
@@ -27,6 +26,8 @@ services:
                   DB_USER: ${postgres_user}
                   DB_PASSWORD: ${postgres_password}
             volumes:
-            - /config/config.json:/config/config.json
+            - type: bind
+            source: /xchain-indexer-config-to-mount.json
```

Comment:
maybe it's better to use underscores for file names? hyphens sometimes cause problems

Reply:
Fixed